crates/hyperqueue/src/server/client/autoalloc.rs

```diff
@@ -271,3 +464,45 @@ async fn try_submit_allocation(
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::server::client::autoalloc::get_arg_by_key;
```

Comment:
Could you please add a complete parsing example, i.e. copy the output from PBS and put it into a test. This would also helps others to understand how does the PBS output look like, since it cannot be really seen from the code now.